crates/fiber/src/unix/arm.rs

```diff
@@ -0,0 +1,82 @@
+// A WORD OF CAUTION
```

Comment:
Given that since PR #3799 Cranelift no longer supports the 32-bit Arm architecture, do we need to keep this file?

Reply:
It's true yeah, although knowing that I originally wrote this file and the x86.S file when I first made the `wasmtime-fiber` crate. My thinking was that I had all the context of fibers booted into my head and the CFI directives and such in particular are really tricky so it was easy for me to bang out some other "maybe one day we'll support these" architectures here. These aren't actually tested on CI at all and there's no actual way to use them unless you use the `wasmtime-fiber` crate directly (and why would you).

Personally I'd err on the side of leaving them here in the sense that I've tested them to make sure they work (even with this PR) and otherwise it's one less thing to add if anyone ever wants to add support for arm/x86. If anyone else spends even a crumb of thought on updating these files though they should definitely be deleted immediately as they're not worth that much.